### DISCUSSION TITLE
Grafana Integration

### DISCUSSION THREAD

**Discussion by ZLLentz** [pcdshub/atef#12] [Ideas]
It may be possible to either:
- Use grafana as an atef config file (suggested by @klauer)
- Use an atef config file to generate grafana pages (suggested by @zllentz)

See:
- https://github.com/weaveworks/grafanalib
- https://pypi.org/project/grafana-api/

**Comment by klauer** [comment]
I had started on a grafanalib tool to create dashboards - with archiver support. Had some local changes to finish up for my upstream PR (see that repo, it's still in draft state there)

**Comment by klauer** [comment]
Attempting to use dashboard as input in #22
